mimic/test/fixtures.py

```diff
@@ -37,21 +37,37 @@ def __init__(self, test_case, root, username, password):
             }
         ))
 
+    def get_tenant_id(self):
+        """
+        Return the tenant id from the service catalog
+        """
+        return self.service_catalog_json['access']['token']['tenant']['id']
+
     def get_service_endpoint(self, service_name, region=''):
         """
-        Return the publicURL for the given service and region. Note that if there are multiple
-        endpoints for a given region, the first will be returned, and if no region is specified,
-        the first endpoint will be returned.
-        :param unicode service_name: The name of the service for which to get an endpoint as
-            listed in the service catalog
-        :param unicode region: The service catalog region of the desired endpoint
+        Return the publicURL for the given service and region. Note that if
+        there are multiple endpoints for a given region, the first will be
+        returned, and if no region is specified, the first endpoint will be
+        returned.
+        :param unicode service_name: The name of the service for which to get
+            an endpoint as listed in the service catalog
+        :param unicode region: The service catalog region of the desired
+            endpoint
```

Comment:
Could you amend the docstring to describe the conditions under which a `KeyError` might be raised?